tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_adam.py

```diff
@@ -8,41 +8,71 @@
 
 import tt_lib as ttl
 import pytest
-from models.utility_functions import (
-    skip_for_wormhole_b0,
-    comp_allclose_and_pcc,
-    comp_pcc,
-)
+from models.utility_functions import skip_for_wormhole_b0, comp_allclose_and_pcc, comp_pcc, is_wormhole_b0
 from loguru import logger
 
+fp32_dest_acc_en = [
+    False,  # for grayskull
+]
+fp32_dest_acc_en_ids = ["fp32_dest_acc_en=False"]
+if is_wormhole_b0:
+    fp32_dest_acc_en.append(True)
+    fp32_dest_acc_en_ids.append("fp32_dest_acc_en=True")
+
+
+def create_tt_tensor(tensor, device):
+    ret = (
+        ttl.tensor.Tensor(
+            tensor,
+            ttl.tensor.DataType.BFLOAT16,
+        )
+        .to(ttl.tensor.Layout.TILE)
+        .to(device)
+    )
+
+    return ret
+
+
+def get_compute_kernel_options(fp32_dest_acc_en):
+    if fp32_dest_acc_en is None:
+        return None
+
+    if is_wormhole_b0:
+        compute_kernel_config = ttl.tensor.WormholeComputeKernelConfig(
+            math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            math_approx_mode=False,
+            fp32_dest_acc_en=fp32_dest_acc_en,
+            packer_l1_acc=False,
+        )
+    else:
+        # Grayskull doesn't support fp32 but test passing a GS config is ok
+        compute_kernel_config = ttl.tensor.GrayskullComputeKernelConfig(
+            math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            math_approx_mode=True,
+        )
+    return compute_kernel_config
```

Comment:
```suggestion
from tests.tt_eager.python_api_testing.unit_testing.misc.test_utils import (
    get_compute_kernel_options,
    compute_kernel_options,
    compute_kernel_ids,
)
```
You can use test_utils package.

Reply:
Done.